pkg/scheduler/eventhandlers.go

```diff
@@ -176,11 +176,14 @@ func (sched *Scheduler) addPodToSchedulingQueue(obj interface{}) {
 }
 
 func (sched *Scheduler) updatePodInSchedulingQueue(oldObj, newObj interface{}) {
-	pod := newObj.(*v1.Pod)
-	if sched.skipPodUpdate(pod) {
+	oldPod, newPod := oldObj.(*v1.Pod), newObj.(*v1.Pod)
+	if oldPod.ResourceVersion == newPod.ResourceVersion {
```

Comment:
adding a comment here explaining when this could happen would be great.

Reply:
Updated.

Reply:
updated.